packages/x11/driver/xf86-video-nvidia/package.mk

```
@@ -38,51 +38,83 @@ make_target() {
 }
 
 makeinstall_target() {
+  # Linux kernel modules
+  mkdir -p ${INSTALL}/usr/lib/nvidia
+    cp -P kernel/nvidia.ko ${INSTALL}/usr/lib/nvidia
+  mkdir -p ${INSTALL}/$(get_full_module_dir)/nvidia
+    ln -sf /var/lib/nvidia.ko      ${INSTALL}/$(get_full_module_dir)/nvidia/nvidia.ko
+    cp -P kernel/nvidia-uvm.ko     ${INSTALL}/$(get_full_module_dir)/nvidia
+    cp -P kernel/nvidia-modeset.ko ${INSTALL}/$(get_full_module_dir)/nvidia
+
+  # X driver
   mkdir -p ${INSTALL}/${XORG_PATH_MODULES}/drivers
     cp -P nvidia_drv.so ${INSTALL}/${XORG_PATH_MODULES}/drivers/nvidia-main_drv.so
     ln -sf /var/lib/nvidia_drv.so ${INSTALL}/${XORG_PATH_MODULES}/drivers/nvidia_drv.so
 
+  # GLX extension module for X
   mkdir -p ${INSTALL}/${XORG_PATH_MODULES}/extensions
   # rename to avoid conflicts with X.Org-Server module libglx.so
     cp -P libglxserver_nvidia.so.${PKG_VERSION} ${INSTALL}/${XORG_PATH_MODULES}/extensions/libglx_nvidia.so
 
+  # Xorg config
   mkdir -p ${INSTALL}/etc/X11
     cp ${PKG_DIR}/config/*.conf ${INSTALL}/etc/X11
 
+  # GLX
   mkdir -p ${INSTALL}/usr/lib
-    cp -P libnvidia-glcore.so.${PKG_VERSION} ${INSTALL}/usr/lib
-    cp -P libnvidia-glsi.so.${PKG_VERSION} ${INSTALL}/usr/lib
-    cp -P libnvidia-ml.so.${PKG_VERSION} ${INSTALL}/usr/lib
-    ln -sf /var/lib/libnvidia-ml.so.1 ${INSTALL}/usr/lib/libnvidia-ml.so.1
-    cp -P libnvidia-tls.so.${PKG_VERSION} ${INSTALL}/usr/lib
     cp -P libGLX_nvidia.so.${PKG_VERSION} ${INSTALL}/usr/lib/libGLX_nvidia.so.0
 
-  mkdir -p ${INSTALL}/$(get_full_module_dir)/nvidia
-    ln -sf /var/lib/nvidia.ko ${INSTALL}/$(get_full_module_dir)/nvidia/nvidia.ko
-    cp -P kernel/nvidia-uvm.ko ${INSTALL}/$(get_full_module_dir)/nvidia
-    cp -P kernel/nvidia-modeset.ko ${INSTALL}/$(get_full_module_dir)/nvidia
+  # GLVND
+  mkdir -p ${INSTALL}/usr/share/glvnd/egl_vendor.d
+    cp -p 10_nvidia.json ${INSTALL}/usr/share/glvnd/egl_vendor.d
 
-  mkdir -p ${INSTALL}/usr/lib/nvidia
-    cp -P kernel/nvidia.ko ${INSTALL}/usr/lib/nvidia
+  # OpenGL / EGL
+  mkdir -p ${INSTALL}/usr/lib
+    cp -p libEGL_nvidia.so.${PKG_VERSION}  ${INSTALL}/usr/lib/
+    ln -sf libEGL_nvidia.so.${PKG_VERSION} ${INSTALL}/usr/lib/libEGL_nvidia.so.0
+    ln -sf libEGL_nvidia.so.0              ${INSTALL}/usr/lib/libEGL_nvidia.so
+
+  # OpenGL core
+  mkdir -p ${INSTALL}/usr/lib
+    cp -p libnvidia-eglcore.so.${PKG_VERSION}  ${INSTALL}/usr/lib/
+    cp -P libnvidia-glcore.so.${PKG_VERSION}   ${INSTALL}/usr/lib
+    cp -p libnvidia-glsi.so.${PKG_VERSION}     ${INSTALL}/usr/lib
```

```suggestion
    cp -p libnvidia-eglcore.so.${PKG_VERSION} \
          libnvidia-glcore.so.${PKG_VERSION} \
          libnvidia-glsi.so.${PKG_VERSION} \
          ${INSTALL}/usr/lib
```

or less nice (imho)

`cp -p {libnvidia-eglcore.so.${PKG_VERSION},libnvidia-glcore.so.${PKG_VERSION},libnvidia-glsi.so.${PKG_VERSION}} ${INSTALL}/usr/lib`

---

"problem" is that NVIDIA tends to add, alter, rename or whatever libs in between driver versions & it's more convenient to just copy paste a line to add a lib